docs/index.rst

```diff
@@ -103,6 +103,7 @@ Computations and utilities
 
    cosmology/index
    convolution/index
+   utils/iers
```

Comment:
Should this be in the "Nuts and bolts" section instead?

Reply:
I thought about that, but I think of Nuts and bolts as pure infrastructure like logging and downloading files. This feels more like science and in fact Earth rotation is a science discipline on par with cosmology.

Reply:
OK, fair enough!